ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.ts

```diff
@@ -59,7 +59,7 @@ export default class DestinationsCreateForm extends Component<Args> {
   @waitFor
   *save(event: Event) {
     event.preventDefault();
-
+    this.error = '';
```

Comment:
Came across this while testing and if the same error was returned twice the banner would remain unchanged and it kind of looked like nothing happened. Removing it while the request is in flight reaffirms that the action did indeed trigger.

Reply:
Good catch! I added this to the sync form, and must have missed it here!